portfolio/src/main/webapp/scripts/overlay.js

```diff
@@ -4,38 +4,41 @@ let activeOverlay = null;
 /**
  * Decides wether to show or hide the overlay.
  */
-function toggleOverlay(section){
-  if(isShowingOverlay) (hideOverlay(activeOverlay));
-  if(activeOverlay === section) (activeOverlay = null);
-  else{
+function toggleOverlay(section) {
+  if (isShowingOverlay) {
```

Comment:
[Optional] I like to add an blank line when I have an 'if' followed by another 'if' and then 'else' to make it easier to see that the 'if' on line 8 doesn't match up with the 'else' on line 13.